core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -24,9 +27,11 @@
     private final EncodingManager encodingManager;
     private final int motorVehicleInt;
 
+    private final double DELTA = 0.1;
+
     public DataFlagEncoderTest() {
         encoder = new DataFlagEncoder();
-        encodingManager = new EncodingManager(encoder);
+        encodingManager = new EncodingManager(Arrays.asList(encoder), 8);
```

Comment:
Got it now. I've just finished a commit that makes encoders for road attributes configurable. Will merge with Peter's change before push.